src/System.Drawing.Common/tests/RegionTests.cs

```diff
@@ -0,0 +1,1930 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
That's going to be tricky. Basically these tests aren't really entirely from Mono - I'm just using their test data in amongst other areas